app/controllers/application_controller.rb

```diff
@@ -1,9 +1,27 @@
+
+
 class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
 
-  def  aaa
-      1 + 1
+  def  aaa (a,b,c, d , e ,      f , g , h)     
```

Comment:
Unused method argument - `a`. If it's necessary, use `_` or `_a` as an argument name to indicate that it won't be used.<br>Unused method argument - `b`. If it's necessary, use `_` or `_b` as an argument name to indicate that it won't be used.<br>Unused method argument - `c`. If it's necessary, use `_` or `_c` as an argument name to indicate that it won't be used.<br>Unused method argument - `d`. If it's necessary, use `_` or `_d` as an argument name to indicate that it won't be used.<br>Unused method argument - `f`. If it's necessary, use `_` or `_f` as an argument name to indicate that it won't be used.<br>Unused method argument - `g`. If it's necessary, use `_` or `_g` as an argument name to indicate that it won't be used.<br>Unused method argument - `h`. If it's necessary, use `_` or `_h` as an argument name to indicate that it won't be used.<br>Avoid parameter lists longer than 5 parameters.<br>Unnecessary spacing detected.<br>Space missing after comma.<br>Do not put a space between a method name and the opening parenthesis.<br>Space found before comma.<br>Trailing whitespace detected.